### PR TITLE
Use libraries v35

### PR DESCRIPTION
**Pull request by janisozaur**

### CODE REVIEW

File: src/openrct2-ui/drawing/engines/SoftwareDrawingEngine.cpp [hunk 1]
```diff
@@ -121,32 +121,20 @@ class SoftwareDrawingEngine final : public X8DrawingEngine
             SDL_UnlockSurface(_surface);
         }
 
-        // Copy the surface to the window
-        if (gConfigGeneral.WindowScale == 1 || gConfigGeneral.WindowScale <= 0)
```
Comment (ZehMatt) [inline src/openrct2-ui/drawing/engines/SoftwareDrawingEngine.cpp:125]: Why did you remove this branch? Now there is always an extra copy.

Comment (janisozaur) [reply src/openrct2-ui/drawing/engines/SoftwareDrawingEngine.cpp:125]: This was causing issues on highdpi on macos. See https://discord.com/channels/264137540670324737/691736850980667514/1227237592743940146. I am aware there is one more copy, but without access to the hardware in question I cannot fix it another way

Comment (janisozaur) [reply src/openrct2-ui/drawing/engines/SoftwareDrawingEngine.cpp:125]: I can guard it for macos only

Comment (janisozaur) [reply src/openrct2-ui/drawing/engines/SoftwareDrawingEngine.cpp:125]: Filed https://github.com/OpenRCT2/OpenRCT2/issues/21772 for addressing this properly - but I cannot do this myself due to lack of hardware required to reproduce it

Comment (ZehMatt) [reply src/openrct2-ui/drawing/engines/SoftwareDrawingEngine.cpp:125]: I see, fair enough.

**Review by ZehMatt — APPROVED** [review]
